strongback/docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Houston docker file
+# Builds an ubuntu base with liftoff for packaging debian applications
```

Comment:
It seemed like a good idea at the time, as extra protection and isolation. This means that strongback would have to be ran on a debian based system (which shouldn't be a problem now, but I'm not a future teller). If you believe liftoff is good enough by itself I can remove the docker dependency.

We will also need the server to have liftoff installed with the houston user having sudo access to that command.